src/components/layout.js

```diff
@@ -38,8 +57,8 @@ const Layout = ({ children }) => {
       onRedirectCallback={onRedirectCallback}
     >
       <Grommet theme={theme} themeMode="dark">
-        <Box background="black">
-          <AppHeader appName="coding earth" />
+        <Box background={topBackground}>
+          <AppHeader appName="coding earth" currentPath={currentPath} />
```

Comment:
not sure what you mean with "currentPath" but you can get the current path by @reach-router on any component: https://reach.tech/router/api/useLocation